File: 2.cxx
/*
 * 2.cxx
 * 
 * Copyright 2019  <>

 */


#include <iostream>
using namespace std;
int main(int argc, char **argv)
{
    
    
    int a,b,c;
    cout << "Podaj pierwsza cyfre";
    cin >> a;
    cout << "Podaj druga cyfre";
    cin >> b;
    cout << "Podaj trzecia cyfre";
    cin >> c;
    cout << "To pierwsza:" << a << endl;
    cout << "To Druga:" << b << endl;
    cout << "To Trzecia:"<< c << endl;
    
    
    if(a>b)
    {
        if(a>c)
        cout << "Pierwsza iczba jest największa:" << a;
        
        else
        {
            
                cout << "Trzecia cyfra jest największa:" << c;
        }
        
        
        }
    if(b>a)
    {
        if(b>c)
        cout << "Druga cyfra jest największa:" << b;
        
        else
        {
            
                cout << "Trzecia cyfra jest największa:" << c;
        }
        
        }    
    
    

    
	
	return 0;
}

File: 2.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  2.py
#  



def main(args):
    
    a = int(input("Podaj 1. liczbę: "))
    b = int(input("Podaj 1. liczbę: "))
    c = int(input("Podaj 1. liczbę: "))
    print ("Pierwsza cyfra:",a);
    print ("Druga cyfra:",b);
    print ("Trzecia cyfra:",c);
    
    if a>b:
        if a>c:
            print ("Pierwsza Cyfra jest największa" , a);
        else:
            print("Trzecia Cyfra jest największa", c);
    if b>a:
        if b>c:
            print("Druga Cyfra jest największa", b);
        else:
            print("Trzecia Cyfra jest największa", c);    
            

          
        
                
    
    
    
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
